my-app/src/components/add.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+class Add extends React.Component {
+  constructor (props) {
```

Comment:
Remove space after the constructor keyword, As You are not giving space after any function name. Follow the same pattern everywhere